example_test.go

```diff
@@ -94,6 +94,9 @@ func ExampleFirebase_Watch() {
 		log.Printf("Type: %s\n", event.Type)
 		log.Printf("Type: %s\n", event.Path)
```

Comment:
While we're here, let's update L95 and L96 with the attributes they're printing. Right now, they both say `Type: `